game-headed/src/main/resources/org/triplea/game/client/ui/javafx/fxml/TripleADownload.fxml

```diff
@@ -1,50 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's good to see we're consolidating/unifying code, seems very much like a good step in the right direction.